Virtual Librarian/VirtualLibrarian.API/Controllers/BooksController.cs

```diff
@@ -36,7 +35,7 @@ public IHttpActionResult GetAvailableBooksSorted()
         [Route("GetAvailableBookCopies")]
```

Comment:
Don't use `Get` in the route. The `HttpGet` verb(attribute) should say that explicitly

Reply:
So it should look like `GET:   /AvailableBookCopies`

Reply:
Same in all other routes